mne/io/kit/tests/test_kit.py

```diff
@@ -185,4 +186,24 @@ def test_hsp_elp():
     assert_array_almost_equal(pts_elp_in_dev, pts_txt_in_dev, decimal=5)
 
 
+def test_decimate():
+    """Test decimation of digitizer headshapes with too many points. """
+    # create fake head points from sphere, and scale to similar numbers as hsp
+    hsp = _get_ico_surface(5)['rr']
+    hsp /= 10
+
+    # read in raw data using spherical hsp, and extract new hsp
+    raw = read_raw_kit(sqd_path, mrk_path, elp_txt_path, hsp)
+    hsp_dec = np.array([dig['r'] for dig in raw.info['dig']])
+
+    # with 10242 points and decimation at 5 mm, hsp_dec should be over 5000
+    assert len(hsp_dec) > 5000
+    # should have similar size, distance from center
+    dist = np.sqrt(np.sum((hsp - np.mean(hsp, axis=0))**2, axis=1))
+    dist_dec = np.sqrt(np.sum((hsp_dec - np.mean(hsp_dec, axis=0))**2, axis=1))
+    hsp_rad = np.mean(dist)
+    hsp_dec_rad = np.mean(dist_dec)
+
+    assert_array_almost_equal(hsp_rad, hsp_dec_rad, decimal=3)
```

Comment:
although in this case it might actually matter pretty soon
```
In [3]: from numpy.testing import assert_array_almost_equal

In [4]: from nose.tools import assert_almost_equal

In [5]: timeit assert_array_almost_equal(1, 1)
10000 loops, best of 3: 75.1 µs per loop

In [6]: timeit assert_almost_equal(1, 1)
The slowest run took 66.25 times longer than the fastest. This could mean that an intermediate result is being cached.
10000000 loops, best of 3: 151 ns per loop
```